devicon.json

```
@@ -8199,6 +8199,35 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "wasm",
+        "altnames": ["WebAssembly"],
```

Adjusting array structure.
```suggestion
        "altnames": [
            "WebAssembly"
        ],
```